### PR TITLE
MCOL-5438 COUNT() in math causes SEGV

### PR DESCRIPTION
**Pull request by drrtuy**

### CODE REVIEW

**Comment by tntnatbry** [comment]
@drrtuy Please also create this PR for develop.

**Comment by drrtuy** [comment]
@tntnatbry I think you meant develop-2302

File: utils/rowgroup/rowgroup.h [hunk 1]
```diff
@@ -1084,7 +1083,8 @@ inline void Row::getInt128Field(uint32_t colIndex, int128_t& x) const
 
 inline datatypes::TSInt128 Row::getTSInt128Field(uint32_t colIndex) const
 {
-  const int128_t* ptr = reinterpret_cast<int128_t*>(&data[offsets[colIndex]]);;
+  const int128_t* ptr = reinterpret_cast<int128_t*>(&data[offsets[colIndex]]);
+  ;
```
Comment (denis0x0D) [inline utils/rowgroup/rowgroup.h]: Nit remove empty operator

File: utils/dataconvert/dataconvert.h [hunk 2]
```diff
@@ -113,27 +118,6 @@ const int64_t IDB_pow[19] = {1,
                              100000000000000000LL,
                              1000000000000000000LL};
 
-const std::string columnstore_big_precision[20] = {"9999999999999999999",
```
Comment (denis0x0D) [inline utils/dataconvert/dataconvert.h:100]: Interesting, we had a string array in header =)

**Review by denis0x0D — APPROVED** [review]

**Review by denis0x0D — CHANGES_REQUESTED** [review]
MTR tests fails

**Comment by tntnatbry** [comment]
@drrtuy Yes that's what I meant, sorry, develop-23.02.

File: utils/rowgroup/rowgroup.h (hunk 1 above)
Comment (drrtuy) [reply utils/rowgroup/rowgroup.h]: Done

**Review by mariadb-LeonidFedorov — DISMISSED** [review]

**Review by mariadb-LeonidFedorov — APPROVED** [review]

**Pull request merged**